config/manager/config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: numaplane-controller-config
+  namespace: numaplane-system
+data:
+  config.yaml: |
+    clusterName: "CLUSTER_NAME_VALUE" # Replace with the cluster name where numaplane will get deployed.
```

Comment:
Thanks. Did you test `make deploy` to confirm that it gets correct set?

Reply:
yes if you set the proper value in clusterName something like numaplane-cluster ,it works